### ISSUE TITLE
Moving entries bewtween groups doesn't work

### ISSUE THREAD

**Issue by skanskan** [JabRef/jabref#4414]
Hello
I'm using JabRef 4.3.1 on Windows 10 X64.
I'm trying to reorder my entries on groups.
If I select several entries and using the rightmouse menu choose "Move to group" only the first entry is moved.
If I try to drag it instead of move it it copies it (and duplicate the entries)

**Comment by AEgit** [comment]
The issue is more serious in:

JabRef 5.0-dev--snapshot--2018-10-28--master--05047f32a
Windows 10 10.0 amd64 
Java 1.8.0_191

1. Select several entries in the main table
2. Rightclick and use the "Move to group" feature
3. All selected entries disappear from their original group but NONE of them is added to the new target group, i. e. these entries now belong to neither of those groups

Regarding the "drag'n'drop" feature. I think, this is supposed to work like this. In previous versions of JabRef dragging one item to another group, would assign to this group as well, NOT assign it exclusively to this group. So this should work as intended.

**Comment by Siedlerchr** [comment]
This should be fixed thanks to @Ali96kz  in #4439 


We would like to ask you to use a development build from https://builds.jabref.org/master and report back if it works for you. Please remember to make a backup of your library before trying-out this version.

**Comment by Siedlerchr** [comment]
@AEgit  I found and fixed the underlying issue. It is a simple fix which makes the feature working again.
https://builds.jabref.org/fixgroupsmove/

Note that the entry count number on the group does not update, although the move is succesful, but that is another issue

**Comment by Siedlerchr** [comment]
Thanks for the update, will check it out with open entry editor. The crash might be coming from the fact that the move group dialog is still swing and the rest javafx. 
I added the dialog to the list of things to be converted.

**Comment by skanskan** [comment]
Now it seems to work. It moves and copies properly between groups, though it doesn't update the number of entries on each of them.

**Comment by Siedlerchr** [comment]
Yeah, the number update is another issue already filed here . If you switch to another library and back the numbers should be correct again

**Comment by AEgit** [comment]
I can confirm again that this bug is fixed in:

JabRef 5.0-dev--snapshot--2018-11-11--master--50875f967
Windows 10 10.0 amd64 
Java 1.8.0_191

Note, however, that the crashing behaviour reported in https://github.com/JabRef/jabref/issues/4414#issuecomment-435709161 persists (I was not sure, whether comment https://github.com/JabRef/jabref/issues/4414#issuecomment-437432632 referred to the main topic of this thread or the crashing behaviour I reported earlier on).
